src/main/java/filter/expression/Qualifier.java

```diff
@@ -160,6 +193,15 @@ public static boolean process(IModel model, FilterExpression expr, TurboIssue is
         return Optional.of(allMilestones.size());
     }
 
+    /**
+     * Checks whether a milestone still has open issues or it is not due yet.
+     */
+    private static boolean isRelevantMilestone(TurboMilestone milestone) {
```

Comment:
'Relevant' to what? This method name cannot be explained without referring to the caller. That should not be the case. A method should be clear on its own.